lib/appliance_console/external_httpd_configuration.rb

```diff
@@ -276,6 +178,27 @@ def host_reachable?(host, what = "Server")
         say("Succeeded.")
         true
       end
+
+      def cp_template(file, src_dir, dest_dir)
+        src_path = path_join(src_dir, file)
+        if src_path.to_s.include?(".erb")
+          dest_path = path_join(dest_dir, file.gsub(".erb", ""))
+          File.write(dest_path, ERB.new(File.read(src_path), nil, '-').result(binding))
+        else
+          FileUtils.cp src_path, dest_dir
+        end
+      end
+
+      def rm_file(file, dir)
+        path = path_join(dir, file)
+        File.delete(path) if File.exist?(path)
+      end
+
+      def path_join(*args)
```

Comment:
what about storing src_dir and dest_dir as a Pathname.

then your code would just be `src_dir.join(file)`